src/mesh/difops.cxx

```diff
@@ -239,7 +239,7 @@ const Field3D Div_par(const Field3D &f, const Field3D &v) {
 
 const Field3D Div_par_flux(const Field3D &v, const Field3D &f, CELL_LOC outloc, DIFF_METHOD method) {
   Coordinates *metric = v.getCoordinates(outloc);
-  return -metric->Bxy*FDDY(v, f/metric->Bxy, outloc, method)/sqrt(metric->g_22);
+  return metric->Bxy*FDDY(v, f/f.getCoordinates()->Bxy, outloc, method)/sqrt(metric->g_22);
```

Comment:
There's a sign change here -- is that intentional?

Reply:
Yes, it's intentional. I think at some point this sign error was covering the one fixed in 41d05ad89252993a271aa411265bf31857a77e90. The sign here is correct (I think) and consistent with both the fixed U1 method, and the other methods (at least I've tested C2).